libs/api-interfaces/src/lib/aggregate-views/view-models/resources/term.view-model.interface.ts

```diff
@@ -18,6 +18,10 @@ export interface ITermViewModel extends IBaseResourceViewModel {
         allowedGroupIds: string[];
     };
 
+    /**
+     * These should not be here. These interfaces are for specifying the contract
```

Comment:
Can we revisit this now? This interface is meant provide a constraint on the query service (via controller and response interceptor) to satisfy a contract with the client. We can't send methods over the wire (without rehydrating instances anyway), these shouldn't be here. Perhaps we want to program to `BaseViewModel` wherever this part of the interface is used?

Reply:
I've removed them. Let's make sure that doesn't break the build. As far as I could until these are now unused after responding to other PR suggestions.